### DISCUSSION TITLE
When can we expect to have minor release which includes newer crypto package

### DISCUSSION THREAD

**Discussion by CodingBurak** [spf13/viper#1963] [Q&A]
When can we expect to have minor release which includes newer crypto package https://github.com/spf13/viper/commit/54f2089833b65fa556a510957197de2609059147 ? As this has a critical CVE reported here: https://www.cve.org/CVERecord?id=CVE-2024-45337

**Comment by sagikazarmark** [comment]
A release is really due now, but I can't say for sure. But you should be able to use an updated version of the crypto package in your projects.

**Reply by CodingBurak** [reply]
> A release is really due now, but I can't say for sure. But you should be able to use an updated version of the crypto package in your projects.

So we dont have a direct dependency to crypto itself, but have a dependency to viper. I think I cant force viper to use the updated crypto package.

**Reply by sagikazarmark** [reply]
Yes you can: just make sure to have an updated version in your go.mod file.
